yt/go/yt/interface.go

```diff
@@ -1261,6 +1261,30 @@ type LookupRowsOptions struct {
 	*TransactionOptions
 }
 
+type MultiLookupSubrequest struct {
+	Path                ypath.Path
+	KeepMissingRows     *bool
+	EnablePartialResult *bool
+	UseLookupCache      *bool
+	Columns             []string
+	Keys                []any
+}
+
+type TabletReadOptions struct {
+	ReadFrom                  *TabletReadKind
+	CachedSyncReplicasTimeout *uint64
+}
+
+type MultiLookupRowsOptions struct {
+	Timestamp          *Timestamp
+	RetentionTimestamp *Timestamp
+	TabletReadOptions  *TabletReadOptions
+	ReplicaConsistency *ReplicaConsistency
+	MultiplexingBand   *MultiplexingBand
```

Comment:
I propose to extract MultiplexingBand into [MultiplexingBandOptions](https://github.com/ytsaurus/ytsaurus/blob/main/yt/yt/client/api/client_common.h#L29) (similar to the C++ API, perhaps this is done for future extensibility) and embed it as well.

Reply:
Done